PsevdoTwwiterApp/src/main/java/servlets/CheckServlet.java

```diff
@@ -0,0 +1,23 @@
+package servlets;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import com.google.gson.Gson;
+
+public class CheckServlet extends HttpServlet {
+
+            boolean success;
+
+            @Override
+            protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+
+                CheckServlet s=new CheckServlet();
```

Comment:
Can't figure out why are you creating another instance of `CheckServlet` from within the Check Servlet? This at least currupts the ServletContainer logic (one servlet instance per container)